apps/native/app/(tabs)/wallet/index.tsx

```diff
@@ -17,8 +28,29 @@ export default function Wallet() {
       <ScrollView
         style={styles.scrollView}
         showsVerticalScrollIndicator={false}>
-        <WalletList wallets={mockWallets} onWalletPress={handleWalletPress} />
+        <View style={styles.header}>
+          <Text style={styles.headerTitle}>Wallet</Text>
+          <Text style={styles.headerSubtitle}>Wallet/Exceeded limit</Text>
+        </View>
```

Comment:
Let’s remove this header block; it’s no longer in the latest version of the main branch.